.github/actions/fetch_ctk/action.yml

```diff
@@ -97,7 +98,13 @@ runs:
         CTK_CACHE_COMPONENTS="${CTK_CACHE_COMPONENTS//,,/,}"
         # Get headers and shared libraries in place
         for item in $(echo $CTK_CACHE_COMPONENTS | tr ',' ' '); do
-            populate_cuda_path "$item"
+            ctk_version="${{ inputs.cuda-version }}"
+            if [[ "$item" == "cuda_sanitizer_api" ]]; then
+              # Always use latest CTK for cuda_sanitizer_api
+              # FIXME: Automatically track latest CTK version
+              ctk_version="12.8.1"
```

Comment:
Since we have the CUDA version encoded in the cache name, I prefer to not have ambiguity. Also, compute-sanitizer does not work across major versions. We don't need to fix anything for CUDA 11. 
```suggestion
              CUDA_MAJOR="$(cut -d '.' -f 1 <<< ${{ inputs.cuda-version }})"
              if [[ "$CUDA_MAJOR" == "12" ]]; then
                # TODO: Automatically track latest CTK minor version
                ctk_version="12.8.0"
              fi
```